src/index.jsx

```
@@ -26,12 +27,24 @@ async function enableMocking() {
   });
 }
 
+const offlineStorageConfig = {
+  type: "indexedDB",
+  name: import.meta.env.VITE_INDEXED_DB_NAME || "radfish_dev",
```

Recalling what Jesse had mentioned, I think we should avoid setting a default db name and instead let it error out if they don't supply one.

---

Along with provide a warning as to what's missing